client-side/src/pages/SignupPage.tsx

```diff
@@ -19,18 +19,18 @@ const SignupForm: React.FC<SignupFormProps> = ({ onSignup }) => {
   };
 
   return (
-    <form onSubmit={handleSubmit}>
+    <form onSubmit={handleSubmit} className='auth-container'>
       <h2>Sign Up</h2>
       <label>
-        Email:
+        Email: <br />
         <input type="email" value={email} onChange={(event) => setEmail(event.target.value)} required />
       </label>
       <label>
-        Password:
+        Password: <br />
         <input type="password" value={password} onChange={(event) => setPassword(event.target.value)} required />
       </label>
       <label>
-        Confirm Password:
+        Confirm Password: <br />
```

Comment:
There are alternative ways to add a new line without having a `<br>` in the label. Try using some css for that.